LobbyCompatibility/LobbyCompatibility.csproj

```diff
@@ -32,24 +32,31 @@
 
     <!-- NuGet Package Info -->
     <PropertyGroup>
-        <Authors>MaxWasUnavailable,legoandmars,Xilophor</Authors>
-        <PackageId>LethalCompany.LobbyCompatibility</PackageId>
+        <PackageId>BMX.LobbyCompatibility</PackageId>
+        <Title>Lobby Compatibility</Title>
```

Comment:
Where is this shown, out of curiosity? 
Should we change it to just `LobbyCompatibility` maybe?

That's how we've been referring to the mod, and I think it helps feel a little less generic without the space

Reply:
I guess [microsoft docs ](https://learn.microsoft.com/en-us/nuget/reference/msbuild-targets#pack-target) say `A human-friendly title of the package, typically used in UI displays as on nuget.org and the Package Manager in Visual Studio.`

Personally I think LobbyCompatibility would make more sense